src/shared/components/person/person-details.tsx

```diff
@@ -142,7 +258,11 @@ export class PersonDetails extends Component<PersonDetailsProps, any> {
             postListingMode="small_card"
             showCommunity
             crossPosts={[]}
-            showBody={"preview"}
+            showBody={getShowBodyMode(this.state.expandedPostIndices, idx)}
+            isExpanded={isExpanded}
+            isHighlighted={isHighlighted}
+            postIndex={idx}
```

Comment:
This shouldn't be necessary, you're already passing the entire postView which has the id.